lib/open_api_spex/plug/swagger_ui.ex

```diff
@@ -127,18 +128,23 @@ defmodule OpenApiSpex.Plug.SwaggerUI do
   """
 
   @impl Plug
-  def init(path: path), do: %{path: path}
+  def init(opts) when is_list(opts) do
+    opts
+    |> Enum.into(%{})
+    |> Map.put_new(:path, "/api/openapi")
```

Comment:
I'd prefer not to have a default for `path` here. 

Perhaps a @doc attribute would be enough to document the available options, and which are required?